### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by LeviRemi**
Finished assignment Front End. Please review.

### CODE REVIEW

File: cst438hw3/src/main/resources/templates/city_show.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <meta charset="ISO-8859-1">
 <title>City</title>
 </head>
-<body style="font-family: Verdana; background: #E9ECEF">
+<body style="font-family: Verdana; background: #E9ECEF; margin: 2%;">
```
Comment (jcfleischli) [inline cst438hw3/src/main/resources/templates/city_show.html:7]: I like the small style changes you made. Simple, but just added a little to the plain page.

File: cst438hw3/src/main/resources/templates/city_show.html [hunk 2]
```diff
@@ -33,8 +33,22 @@ <h1 th:inline="text">City</h1>
 		</tr>
 		
 		</table>
-		
-		
+		<br><br>
+		<p>For a fabulous weekend get away for 2 to [[${city.name}]] sign up here:</p>
+		<form action="#" th:action="@{/cities/reservation}" method="post">
```
Comment (jcfleischli) [inline cst438hw3/src/main/resources/templates/city_show.html:38]: I believe your form can be slightly simplified by not needing the hidden input. The city name can be passed through the form th:action with this: th:action="@{/cities/reservation(city=${cityInfo.name})}"

File: cst438hw3/src/main/resources/templates/request_reservation.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html xmlns:th="www.thymeLeaf.org">
+<head>
+<meta charset="ISO-8859-1">
+<title>Reservation Requested</title>
+</head>
+<body>
+	<p>The details of your weekend get away at the <strong>[[${level}]]</strong> level to <strong>[[${city}]]</strong> are being emailed to <strong>[[${email}]]</strong></p>
```
Comment (jcfleischli) [inline cst438hw3/src/main/resources/templates/request_reservation.html:8]: Values for level, city and email were all passed and displayed properly.

File: cst438hw3/src/main/resources/templates/city_show.html (hunk 2 above)
Comment (LeviRemi) [reply cst438hw3/src/main/resources/templates/city_show.html:38]: Wow that is simpler. Thank you!

**Pull request merged**